src/cascadia/TerminalConnection/ConptyConnection.h

```diff
@@ -32,7 +32,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
 
         void Start();
         void WriteInput(const hstring& data);
-        void Resize(uint32_t rows, uint32_t columns);
+        void Resize(til::CoordType rows, til::CoordType columns);
```

Comment:
Isn't this part of the `idl` API? Are we changing the definition of `ITerminalConnection`? Where does narrowing happen, and can we detect it?

Reply:
Oh thanks for catching this! The narrowing should happen in cppwinrt's generated wrapper code.
[C4365](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4365?view=msvc-170) is off by default, but should catch this issue.